prac_03/Do this now.py

```diff
@@ -12,7 +12,12 @@
 # print(things[-2])
 
 
-names = ["Bob", "jimbo", "sally"]
-for i in range(len(names)):
-    names[i] = names[i].title()
-print(names)
```

Comment:
Always remember to add a blank line at the end, it clears up that PEP8 underline